utils/rpms/daos.spec

```diff
@@ -401,6 +407,11 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r daos_agent
 %{_libdir}/*.a
 
 %changelog
+* Wed Nov 11 2020 Tom Nabarro <tom.nabarro@intel.com> 1.1.1-7
```

Comment:
It's too late now, but as a note for the future, this PR didn't actually bump the `Release:` in this spec file.

Reply:
fixed, thanks